### PR TITLE
feat: add option sheet and metadata

### PR DESCRIPTION
**Pull request by janrtvld**
https://github.com/user-attachments/assets/2ebe7c58-55b2-4db9-afe1-2d23e3307ead

### CODE REVIEW

**Comment by janrtvld** [comment]
Alignd it differently so there is more space on the screen now. I'm still rethinking this screen. I don't think it's very clear the reason is supplied from the issuer. Also, the card structure is a bit too much like the NL wallet, but I do like it 😕. Ideas are welcome :).

IMG_BC5D368A6A5C-1

File: apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx [hunk 1]
```diff
@@ -47,41 +75,90 @@ export function FunkeRequestedAttributesDetailScreen({
     return null
   }
 
+  const displayedMetadata = useMemo(() => {
```
Comment (TimoGlastra) [inline apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx:78]: can't we just not render undefined fields? this feels a bit redundant

File: packages/app/src/utils/formatCredentialSubject.ts [hunk 2]
```diff
@@ -53,9 +53,6 @@ export function formatCredentialSubject(
 
   const isArray = Array.isArray(subject)
   for (const [key, value] of Object.entries(subject)) {
-    // omit id and type
-    if (key === 'id' || key === 'type') continue
```
Comment (TimoGlastra) [inline packages/app/src/utils/formatCredentialSubject.ts:57]: this will restul in weird behaviour for w3c creds which rely on this

**Review by TimoGlastra — APPROVED** [review]

File: apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx (hunk 1 above)
Comment (janrtvld) [reply apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx:76]: Hmm the formatDate is needed though

Comment (TimoGlastra) [reply apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx:76]: Shouldn't we do that in the getCredentialForDisplay?

Comment (janrtvld) [reply apps/easypid/src/features/share/FunkeRequestedAttributesDetailScreen.tsx]: 👍

**Pull request merged**